src/ast/ddl.rs

```diff
@@ -600,6 +600,8 @@ pub enum ColumnOption {
         sequence_options: Option<Vec<SequenceOptions>>,
         generation_expr: Option<Expr>,
         generation_expr_mode: Option<GeneratedExpressionMode>,
+        /// false if 'GENERATED ALWAYS' is skipped (option starts with AS)
+        generated_kw: bool,
```

Comment:
Personal preference, but I would be inclined to expand the name to `generated_keyword` for clarity.